libs/data-mapper-v2/src/components/functionList/styles.tsx

```diff
@@ -0,0 +1,23 @@
+import { makeStyles } from '@fluentui/react-components';
+
+export const useStyles = makeStyles({
+  functionSearchBox: {
+    width: '210px',
+  },
+  functionTree: {
+    backgroundColor: '#E8F3FE',
+  },
+  functionTreeItem: {
+    backgroundColor: '#E8F3FE',
+    paddingLeft: '10px',
+  },
+});
+
+export const FunctionsSVG = () => (
```

Comment:
Ahh I had it in the tsx file initially. Can we move it to it's own images.tsx or something please? styles generally just mean the className or styling we apply